client/my-sites/plugins/notices.jsx

```diff
@@ -821,6 +821,14 @@ class PluginNotices extends Component {
 							}
 						);
 
+					case 'server_error':
+						return translate(
+							'Error activating %(plugin)s on %(site)s because it triggered a fatal error. As a safety measure, the plugin is installed but remains deactivated.',
```

Comment:
does `server_error` always mean it was installed but not activated?

Reply:
I added a `lastStep` to track where the `errorCallback` occurred, if the server error happened during the `doActivate` then this error state should be accurate.